pdh_check/README.md

```diff
@@ -0,0 +1,43 @@
+# Wmi_check Integration
+
+## Overview
+
+Get metrics from wmi_check service in real time to:
+
+* Visualize and monitor wmi_check states
+* Be notified about wmi_check failovers and events.
```

Comment:
Does this replace the existing wmi integration?

Reply:
Nice catch on the copy/paste error. My hope is that it will eventually replace WMI; but initially it will be a new check shipped in addition to wmi